docs/en/changelogs/index.md

```diff
@@ -23,7 +23,6 @@ For information on how to upgrade to newer versions consult the [upgrading](/ins
  * [2.4.7](2.4.7) - 1 February 2012
  * [2.4.6](2.4.6) - 18 October 2011
  * [2.4.10](2.4.10) - 2013-02-19
- * [2.4.9](2.4.9) - 2012-12-04
```

Comment:
How about we make these non-links, but leave the info? I think it's good to have the release history there.